website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -204,15 +204,14 @@ This endpoint creates a destination to synchronize secrets with an Azure Key Vau
 
 - `key_vault_uri` `(string: <required>)` - URI of an existing Azure Key Vault instance.
 
-- `subscription_id` `(string: <required>)` - ID of the target Azure subscription.
```

Comment:
removed subscription_id since it's not actually required.
Made client_id, client_secret, and tenant_id required.

Reply:
@austingebauer are these changes making sense for you? You know Azure more than us.

Should we update some of the docs/comments on the [syncer](https://github.com/hashicorp/secrets-store-syncer/blob/main/stores/azurekv/azurekv.go#L42-L56) & [Vault](https://github.com/hashicorp/vault-enterprise/blob/main/vault/logical_system_sync_ent.go#L721-L727) as well to better document what is required and what is not?